scim2-sdk-common/src/main/java/com/unboundid/scim2/common/utils/Parser.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015-2020 Ping Identity Corporation
+ * Copyright 2015-2021 Ping Identity Corporation
```

Comment:
Instead of updating copyright statements piecemeal, we usually update entire repos at the beginning of each year using Neil's copyright tool. Can you please do that in a separate MR?